executor/show.go

```diff
@@ -600,7 +600,7 @@ func (e *ShowExec) fetchShowCreateTable() error {
 						defaultValBinaryLiteral := types.BinaryLiteral(defaultValStr)
 						fmt.Fprintf(&buf, " DEFAULT %s", defaultValBinaryLiteral.ToBitLiteralString(true))
 					} else {
-						fmt.Fprintf(&buf, " DEFAULT '%s'", format.OutputFormat(defaultValStr))
+						fmt.Fprintf(&buf, " DEFAULT '%s'", format.OutputFormatForEnmu(defaultValStr))
```

Comment:
This only works for `SHOW CREATE TABLE`? Shall we cut off the trailing spaces during creation of the table? i.e, my concern is, when we insert rows into the table, whether `a ` is valid or not?

Reply:
ok, i will try to solve it.